app/scripts/card.js

```diff
@@ -11,32 +11,48 @@ const Session = require('./session.js');
 const database = Firebase.database();
 Vue.use(VueFire);
 
-const route = window.location.pathname;
-
 class Card {
 
     constructor () {
 
         Session.check();
-        this.get();
+
+        const route = window.location.pathname.replace(/^(\/)/g, '');
+        this.create(route);
     }
 
-    get () {
+    create (route) {
+
+        let databaseRef = database.ref(route);
+
         let cards = new Vue({
-            el: '#js-card',
+            el: '#cards',
```

Comment:
What do you think to use data-attributes to components in the future?
You will have a semantical problem when you need to use the same component two or more times in a single page.

An attribute called `data-component` should be good.